DUnitX.TestFramework.pas

```diff
@@ -584,22 +602,6 @@   TDUnitX = class
     function GetReport: string;
   end;
 
-
-  ETestFrameworkException = class(Exception);
```

Comment:
From memory these needed to be in TestFramework so that they would be included when please included DUnitX.TestFramework. This might break some peoples usage of DUnitX.